runtime/gc_base/StringTable.cpp

```diff
@@ -646,6 +646,8 @@ j9gc_createJavaLangString(J9VMThread *vmThread, U_8 *data, UDATA length, UDATA s
 				J9VMJAVALANGSTRING_SET_COUNT(vmThread, result, (I_32) unicodeLength);
 			} else {
 				if (J9VMJAVALANGSTRING_COMPRESSIONFLAG(vmThread, stringClass) == 0) {
+					// jitHookClassPreinitialize will process initialization events for String compression sedeEffectGuards
```

Comment:
Two nitpicks: 
1) VM code never uses `//` comments and always uses `/* */` style comments.
2) `sedeEffectGuards` -> `sideEffectGuards`

Reply:
Uups. Totally forgout about the `/*`. It's been a while. Fixed in 602d0a9